code/game/objects/items/robot/robot_upgrades.dm

```diff
@@ -262,6 +262,16 @@
 	for(var/obj/item/reagent_containers/borghypo/F in R.module.modules)
 		F.emag_act()
 
+/obj/item/borg/upgrade/incision_management
+	name = "medical cyborg IMS upgrade"
+	desc = "An Incision Managment System replacment for the medical cyborg's standard scapel."
```

Comment:
```suggestion
	desc = "An Incision Management System replacment for the medical cyborg's standard scapel."
```

Reply:
Ah thank you for picking it up! English my beloved.